File: VS_SecondLifeGrp6/VS_SecondLifeGrp6.Model/Entities/Photo.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace VS_SLG6.Model.Entities
{
    public class Photo
    {
        public int Id { get; set; }
        public Product Product { get; set; }
        public string Url { get; set; }
    }
}

File: App/app/src/main/java/com/example/secondlife/model/Photo.java
package com.example.secondlife.model;

import java.io.Serializable;

public class Photo implements Serializable {
    private int id;
    private Product product;
    private String url;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public Product getProduct() {
        return product;
    }

    public void setProduct(Product productId) {
        this.product = productId;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}
